catalogue/webapp/pages/works-context.js

```diff
@@ -0,0 +1,248 @@
+// @flow
+import { useContext } from 'react';
+import SearchContext, {
+  type CatalogueQuery,
+  SearchProvider,
+  SearchRouter,
+} from '../components/SearchContext/SearchContext';
+
+type CheckboxesProps = {|
+  options: {| id: string, label: string |}[],
+  name: string,
+  values: string[],
+  setter: (values: string[]) => void,
+|};
+
+const Checkboxes = ({ name, options, values, setter }: CheckboxesProps) => (
+  <fieldset>
+    {options.map(({ id, label }) => (
+      <label key={id}>
+        {label}
+        <input
+          type="checkbox"
+          name={name}
+          value={id}
+          checked={values.indexOf(id) !== -1}
+          onChange={event =>
+            setter(
+              event.currentTarget.checked
+                ? [id, ...values].sort()
```

Comment:
Were we resolved to use `lodash` for sort everywhere after we've seen weirdness with it?

Reply:
I've removed it as I can't think where this might make the most sense, if we order, it feels like it should be at router / context setter.